file_prim/__init__.py

```
@@ -1115,20 +1189,24 @@ def unregister():
+        ExportPRIM.bl_idname, text="Glacier RenderPrimitive (.prim)"
     )
     exportprim_instance.export_scene = False
     exportprim_instance2 = self.layout.operator(
         ExportPRIM.bl_idname,
-        text="Glacier RenderPrimitve (prims, materials, textures, geomentities and collision)",
+        text="Glacier RenderPrimitive (prims, materials, textures, geomentities and collision)",
     )
     exportprim_instance2.export_scene = True
 
 
+def menu_func_import_scenario(self, context):
```

Ruff (0.14.8)

Unused function argument: `context`

(ARG001)